### ISSUE TITLE
HK restarts CH and LB on boot

### ISSUE THREAD

**Issue by GGunnison** [MIT-STARLab/CLICK-A-RPi#11]
New issue found where HK restarts CH and LB on boot, but only once. See attached boot log

[restarts.txt](https://github.com/MIT-STARLab/CLICK-A-RPi/files/6079009/restarts.txt)

**Comment by htomio** [comment]
I've pushed an update to housekeeping but need to explore this further - could you get me the boot log again after pulling the latest code?

**Comment by cierny** [comment]
Resolved with time setting service
